ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Debian.yml

```
@@ -33,10 +33,14 @@
   apt_repository:
     repo: "deb https://adoptopenjdk.jfrog.io/adoptopenjdk/deb {{ ansible_distribution_release }} main"
     update_cache: no
+  when:
+    - (ansible_architecture != "riscv64")
+  tags: patch_update
 
 - name: Add the ubuntu toolchain repository to apt
   apt_repository: repo='ppa:ubuntu-toolchain-r/test' update_cache=no
   when:
+    - (ansible_architecture != "armv7l") and (ansible_architecture != "riscv64")
```

We seem to have `armv7l` here and on the next line
```suggestion
    - ansible_architecture != "riscv64"
```